### ISSUE TITLE
DatabaseException: Error updating 'CVE-2020-36569'

### ISSUE THREAD

**Issue by ben-manes** [dependency-check/dependency-check-gradle#300]
It looks like the internal db needs to use a wider column.

[full log](https://github.com/ben-manes/caffeine/actions/runs/3853202396/jobs/6565987231)

```
> Task :dependencyCheckAggregate
Verifying dependencies for project caffeine
Checking for updates and analyzing dependencies for vulnerabilities
org.owasp.dependencycheck.data.nvdcve.DatabaseException: Error updating 'CVE-2020-36569'
org.owasp.dependencycheck.data.update.exception.UpdateException: org.owasp.dependencycheck.data.nvdcve.DatabaseException: Error updating 'CVE-2020-36569'
	at org.owasp.dependencycheck.data.update.nvd.ProcessTask.processFiles(ProcessTask.java:157)
	at org.owasp.dependencycheck.data.update.nvd.ProcessTask.call(ProcessTask.java:114)
	at org.owasp.dependencycheck.data.update.nvd.ProcessTask.call(ProcessTask.java:41)
	at java.base@17.0.5/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base@17.0.5/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base@17.0.5/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@17.0.5/java.lang.Thread.run(Thread.java:833)
Caused by: org.owasp.dependencycheck.data.nvdcve.DatabaseException: Error updating 'CVE-2020-36569'
	at org.owasp.dependencycheck.data.nvdcve.CveDB.updateVulnerability(CveDB.java:823)
	at org.owasp.dependencycheck.data.update.nvd.NvdCveParser.parse(NvdCveParser.java:114)
	at org.owasp.dependencycheck.data.update.nvd.ProcessTask.importJSON(ProcessTask.java:141)
	at org.owasp.dependencycheck.data.update.nvd.ProcessTask.processFiles(ProcessTask.java:154)
	... 6 more
Caused by: org.h2.jdbc.JdbcBatchUpdateException: Value too long for column "VERSIONENDEXCLUDING CHARACTER VARYING(60)": "'0.0.0-20160722212129-ac0cc4484ad4_before_v0.0.0-20200131131040-063a3fb69896' (75)"; SQL statement:
INSERT INTO software (cveid, cpeEntryId, versionEndExcluding, versionEndIncluding, versionStartExcluding, versionStartIncluding, vulnerable) VALUES (?, ?, ?, ?, ?, ?, ?) [22001-214]
	at org.h2.jdbc.JdbcPreparedStatement.executeBatch(JdbcPreparedStatement.java:1[269](https://github.com/ben-manes/caffeine/actions/runs/3853202396/jobs/6565987231#step:5:279))
	at org.apache.commons.dbcp2.DelegatingStatement.executeBatch(DelegatingStatement.java:241)
	at org.apache.commons.dbcp2.DelegatingStatement.executeBatch(DelegatingStatement.java:241)
	at org.owasp.dependencycheck.data.nvdcve.CveDB.executeBatch(CveDB.java:1248)
	at org.owasp.dependencycheck.data.nvdcve.CveDB.updateVulnerabilityInsertSoftware(CveDB.java:1098)
	at org.owasp.dependencycheck.data.nvdcve.CveDB.updateVulnerability(CveDB.java:816)
	... 9 more
Unable to continue dependency-check analysis.
Unable to update 1 or more Cached Web DataSource, using local data instead. Results may not include recent vulnerabilities.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':dependencyCheckAggregate'.
> Analysis failed.
```

**Comment by KoheiMoroi** [comment]
I have encountered this exception using the latest version `7.4.3`, too.

**Comment by whereispie** [comment]
yeah, got the same problem now. 60 chars is small amount, need 70 + in software column

**Comment by aikebah** [comment]
See also https://github.com/jeremylong/DependencyCheck/issues/5220

A fix is being worked on; the base libraries have been released

**Comment by aikebah** [comment]
Gradle plugin is also available for 7.4.4 (https://github.com/jeremylong/DependencyCheck/issues/5220#issuecomment-1374031615)

**Comment by jeremylong** [comment]
homebrew will be published shortly
